### ISSUE TITLE
thumnail not loading

### ISSUE THREAD

**Issue by AmrutGhatage** [Codewaves/YouTube-Thumbnail-View#2]
1) <meta-data android:name="com.codewaves.youtubethumbnailview.ApiKey"
            android:value="AIzaSyAYAQ03TSJUOtQofk6blvETrZbeMc" />
 2) Written Application class for  ThumbnailLoader.initialize()
 3) load the thumbnails bellow
 thumb2.loadThumbnail("https://www.youtube.com/watch?v=H7jtC8vjXw8", new ImageLoader() {
                    @Override
                    public Bitmap load(String url) throws IOException {
                        return Picasso.with(context).load(url).get();
                    }
                });*/
 4)compile'com.codewaves.youtubethumbnailview:library:0.9.2'
5) 
public class MyApplication  extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        ThumbnailLoader.initialize(getApplicationContext());
    }}
6)Manifest
 <application
        android:name=".MyApplication" .
    .../>

Still not loading thumbnail.. Is any thing missing?

**Comment by Codewaves** [comment]
Have you changed application class in manifest file?

**Comment by AmrutGhatage** [comment]
Yes all ready changed application class in manifest. Updated my question.

**Comment by Codewaves** [comment]
Are you running sample from the library or this is your own project?

**Comment by AmrutGhatage** [comment]
Running on my project. 

    On Wednesday, 13 September 2017 10:02 PM, Sergej Kravcenko <notifications@github.com> wrote:
 

 Are you running sample from the library or this is your own project?—

**Comment by AmrutGhatage** [comment]
Running on my own project

**Comment by AmrutGhatage** [comment]
not executing 
public Bitmap load(String url) throws IOException
Plz provide solution. Thanks in advance.

**Comment by Codewaves** [comment]
Try to add loading listener and check for possible errors using loadThumbnail(String url, ThumbnailLoadingListener listener, ImageLoader imageLoader)
